kubernetes/config-maps/concent/email_settings.py.j2

```diff
@@ -0,0 +1,9 @@
+from ..secrets.secrets import EMAIL_PASSWORD
+
+EMAIL_HOST = "{{ email.host }}"
+EMAIL_PORT = {{ email.port }}
```

Comment:
Please move `RAVEN_CONFIG` to this file too. Anything related to error reporting/Sentry should be here.

Reply:
There is a problem with moving `RAVEN CONFIG`, because we added tag with hard coded name of container.

Reply:
In that case define an extra settings module called `constants.py` with a setting called `KUBERNETES_COMPONENT_NAME`. Then import and use it in all places where we have this hard-coded in settings right now.